artifactory/services/utils/specutils.go

```diff
@@ -91,7 +91,9 @@ func (params *CommonParams) SetPattern(pattern string) {
 }
 
 func (params *CommonParams) SetTarget(target string) {
-	params.Target = target
+	// Target Specifies the target path in Artifactory in the following format: <repository name>/<repository path>, so it cannot start with a slash.
+	// Remove leading slash if exists
+	params.Target = strings.TrimPrefix(target, "/")
```

Comment:
The Target is a public field and can be set directly.
Let's ensure that the code works without this change.

Same comment for the SetUrl in CommonConfigFields.